api/types.go

```diff
@@ -67,7 +67,8 @@ type ChatResponse struct {
 	CreatedAt time.Time `json:"created_at"`
 	Message   Message   `json:"message"`
 
-	Done bool `json:"done"`
+	LoadedMessages []Message `json:"loaded_messages,omitempty"`
```

Comment:
This feels like it belongs in `ShowResponse`, along with the other layers that make up a model

Reply:
I ended up switching this over.